src/documentation/pages/components/date-picker/examples/_calendar-date-time.vue

```diff
@@ -0,0 +1,65 @@
+<template lang="pug">
+  <ComponentExample titleSize="h4" title="Date and Time" id="calendar-date-time" :tabs="exampleTabs">
+    .-d--flex(style="width: 28.75rem;" slot="example")
+      .chi-datepicker.-week-starts-on-sun.-month-starts-on-wed
+        .chi-datepicker__month-row
+          .prev
+            i.chi-icon.icon-chevron-left.-sm(aria-hidden="true")
+          .chi-datepicker__month
+            | November 2018
+          .next
+            i.chi-icon.icon-chevron-right.-sm(aria-hidden="true")
+        .chi-datepicker__day-names
+          .chi-datepicker__week-day(v-for="weekDay in weekDays")
+            | {{weekDay}}
+        .chi-datepicker__days
+          .chi-datepicker__day(v-for="day in Array(30).keys()") {{+day + 1}}
+      .chi-time-picker
+        .chi-time-picker__content
+          .chi-time-picker__hours
+            .chi-time-picker__hour(v-for="hour in hours") {{hour}}
+          .chi-time-picker__minutes
+            .chi-time-picker__minute(v-for="minute in minutes") {{minute}}
+          .chi-time-picker__periods
+            .chi-time-picker__period AM
+            .chi-time-picker__period PM
```

Comment:
I suggest using web components instead of HTML Blueprint code.

```suggestion
    .-d--flex(style="width: 28.75rem;" slot="example")
       chi-date
       chi-time
```